File: ios/RNMBX/RNMBXLightViewManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_REMAP_MODULE(RNMBXLight, RNMBXLightViewManager, RCTViewManager)

// light props
RCT_EXPORT_VIEW_PROPERTY(reactStyle, NSDictionary);

@end

File: ios/RNMBX/RNMBXLightViewManager.swift
@objc(RNMBXLightViewManager)
class RNMBXLightViewManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return false
    }
  
    override func view() -> UIView! {
      let light = RNMBXLight()
      light.bridge = self.bridge
      return light
    }
}
